docs/source/history.rst

```diff
@@ -1,7 +1,7 @@
 History and Audit
 =================
 
-These are records maintained by |st2| of past action executions.  These records include information about what events triggered what rules and then what actions; common uses for this information includes root cause analysis reporting as well as operational control as well as collaboration.
+These are records maintained by |st2| of past action liveactions.  These records include information about what events triggered what rules and then what actions; common uses for this information includes root cause analysis reporting as well as operational control as well as collaboration.
```

Comment:
"These are records maintained by |st2| of past action executions" seems more readable.